Orange/widgets/data/owtable.py

```diff
@@ -76,6 +79,9 @@ def data(self, index, role=Qt.DisplayRole,
                 return (val - dist.min) / (dist.max - dist.min)
             else:
                 return None
+        if role == _Qt_DisplayRole and self._string[index.column()]:
+            val = super().data(index, TableModel.ValueRole)
```

Comment:
I think it would be clearner if you called super with `_Qt_DisplayRole`. (And then don't cast to string.)

If I overlooked something and you need `TableModel.ValueRole`, then add `_ValueRole` to arguments, (see `_BarRole`).